az-digital/az_quickstart
# Closes #1663 New paragraph type for side-by-side Event and News views

## Description

Adds a new "Views Side-by-Side" paragraph type, a new "AZ Side-by-Side Views" view, and new "Row View with Background" view displays for both AZ Event and AZ News.

## Related issues




#1663 

## How to test


Link to Probo site: https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-2583.probo.build
Demo page for testing: https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-2583.probo.build/brian-test-page

1. Install the 'Quickstart Paragraphs - Views Side-by-Side' module (if someone hasn't done this already).
2. In a test page or block, add a new Views Side-by-Side paragraph.
3. For each view:
  a. (optional) Enter a title to appear as a heading above the view.
  b. Select `AZ Side-by-Side` for the view and then select any view display (except Master).
  c. (optional) Add a link to appear as a button underneath the view.
4. Save the page and confirm that the views are showing the items as expected on the page.
5. Edit the page again and test with taxonomy ID arguments for news and events: you can add a single ID or multiple IDs joined with a `+`. The appropriate items will be restricted by those IDs.

### Note about multiple arguments (updated August 30, 2023)
For the new view displays, we may want to use the "Has taxonomy term ID" filter instead of the "Has taxonomy term ID (with depth)" filter. In that case, using a parent taxonomy ID as an argument will only show items that have that specific taxonomy term: it will not filter based on children of that term as well. However, it will allow content editors to use a comma to join IDs (like an 'AND' operation). 


## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
